projecttacoma/synthea
# Exm165 module

# Summary
Tested EXM165 Module and added it to Synthea Modules 

## New Behavior
Synthea repo now contains module which can generate numerator patients for the EXM 165 Measure

## Code Changes
Added the `EXM165.json` file to the src/main/resources/modules directory

# Testing Guidance
- Clear your `output` directory if you have one
- Generate 1000 new patients using the command `./run_synthea -p 1000 -m "EXM16*" `
- Use the vscode global search to search all files in the output directory for "Blood Pressure"
- For any patients that pop up (I got 25/1000), search through the patient bundle to ensure that they have hypertension (FHIR condition) and an encounter (FHIR Encounter) where Blood Pressure was taken and read Diastolic BP < 90 and Systolic BP < 140
- Dowload the EXM165 measure posted in the Slack thread.
- Open fqm-execution, and run `npm run build`
- Then run `node build/cli.js -o "reports" -m {PATH-TO-EXM165-MEASURE} -p {PATH-TO_NUMER-PATIENT}  -s "2021-01-01" -e "2022-01-01"`
- Note, the period start and end may need to be adjusted so that the encounter where the patient had a blood pressure test occurred within the window.
- You should receive a report that lists a measureScore with a value of 1
